docs/source/api/proxy.md

```diff
@@ -17,5 +17,5 @@
 
 ```{eval-rst}
 .. autoconfigurable:: ConfigurableHTTPProxy
-   :members: debug, auth_token, check_running_interval, api_url, command
+   :members:
```

Comment:
Previews:

This PR: https://jupyterhub--4253.org.readthedocs.build/en/4253/api/proxy.html#configurablehttpproxy
Main branch: https://jupyterhub.readthedocs.io/en/latest/api/proxy.html#configurablehttpproxy